turbine/src/cluster_nodes.rs

```diff
@@ -242,13 +242,6 @@ impl RngCore for TurbineRng {
             TurbineRng::ChaCha8(cha_cha8_rng) => cha_cha8_rng.fill_bytes(dest),
         }
     }
-
-    fn try_fill_bytes(&mut self, dest: &mut [u8]) -> Result<(), rand::Error> {
-        match self {
-            TurbineRng::Legacy(cha_cha20_rng) => cha_cha20_rng.try_fill_bytes(dest),
-            TurbineRng::ChaCha8(cha_cha8_rng) => cha_cha8_rng.try_fill_bytes(dest),
-        }
-    }
```

Comment:
Confirmed that this was a required trait method that is no longer needed with new rand